src/otx/v2/cli/cli.py

```diff
@@ -349,8 +349,8 @@ def instantiate_classes(self, subcommand: str) -> None:
         self.model = self.auto_runner.get_model(model={**model_cfg}, num_classes=num_classes)
         # For prediction class
         # NOTE: We'll need to move this somewhere other than here in the next phase.
-        if num_classes is not None and "num_classes" in model_cfg.get("head", {}):
-            model_cfg["head"]["num_classes"] = num_classes
+        if getattr(self.model, "config_dict", None):
+            model_cfg = self.model.config_dict
```

Comment:
Can this be more explicit?
It's weird to see CLI checks if some attribute exists.
```suggestion
        model_cfg = self.model.get_config()
```

Reply:
If it's temporary, let's keep it as is. Let me know your opinion. :)

Reply:
It sounds great, however, currently we don't have a proper model API now. So, `self.model` is a `nn.Module` itself. 
We might need wrapper or APIs for the Models. How about applying this opinion when we bring the Model API?

Reply:
Sure.

Reply:
Thank you for your opinion. I totally agree with you that we do not expose implementation details like this. However, this is temporary solution before we refactor our model api. Currently self.model is just nn.Module, not our model api. So this PR try to add temporary attribute to nn.Module if they change input model api. What do you think?